modules/distr-tracing-removing-tempo-instance-cli.adoc

```diff
@@ -0,0 +1,89 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-tempo-removing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-removing-tempo-instance-cli_{context}"]
+= Removing a {TempoName} instance from the CLI
```

Comment:
For deleting a TempoStack instance, these should be the steps.
* Get the instance of TempoStack created in a project.
`oc get tempostacks.tempo.grafana.com`
* Delete the TempoStack instance.
`oc delete tempostacks.tempo.grafana.com TEMPOSTACK_INSTANCE_NAME`


Example output:
```
$ oc project
Using project "tracing-system" on server "https://api.ikanse-55.qe.gcp.devcluster.openshift.com:6443".
$ oc get pods
NAME                                             READY   STATUS    RESTARTS   AGE
tempo-simplest-compactor-77cc4b9c9f-czlpq        1/1     Running   0          18m
tempo-simplest-distributor-6759b88d99-6j552      1/1     Running   0          18m
tempo-simplest-ingester-0                        1/1     Running   0          18m
tempo-simplest-querier-74d77fc548-2m4cv          1/1     Running   0          18m
tempo-simplest-query-frontend-54b7c465b5-8dn96   2/2     Running   0          18m
$ oc get tempostacks.tempo.grafana.com 
NAME       AGE   TEMPO VERSION   MANAGEMENT
simplest   18m   2.1.1           Managed
$ oc delete tempostacks.tempo.grafana.com simplest 
tempostack.tempo.grafana.com "simplest" deleted
$ oc get pods
No resources found in tracing-system namespace.
```
 
For uninstalling the Tempo operator itself, we can follow the below steps:
*Switch to the project in which Tempo operator is installed.
`oc project openshift-tempo-operator`
*Get the Tempo operator subscription and CSV.
```
$ oc get sub
NAME            PACKAGE         SOURCE           CHANNEL
tempo-product   tempo-product   tempo-registry   stable
```
```
[$ oc get csv | grep -i tempo
tempo-operator.v0.3.0-1            Tempo Operator                                          0.3.0-1    tempo-operator.v0.1.0-6-0.1687199947.p            Succeeded
```
*Delete the Tempo operator subscription and CSV.
```
 $ oc delete sub tempo-product
subscription.operators.coreos.com "tempo-product" deleted
$ oc delete csv tempo-operator.v0.3.0-1
clusterserviceversion.operators.coreos.com "tempo-operator.v0.3.0-1" deleted
```
*Delete the Tempo operator CRDs.
```
oc delete customresourcedefinitions.apiextensions.k8s.io tempostacks.tempo.grafana.com
customresourcedefinition.apiextensions.k8s.io "tempostacks.tempo.grafana.com" deleted
```

Reply:
Saving this for the next PR.